### DISCUSSION TITLE
Question about planning multiple target pose

### DISCUSSION THREAD

**Discussion by jsupratman13** [NVlabs/curobo#73] [Software Q&A] [answered]
Hi, I was wondering if cuRobo can plan for multiple target poses and if so how to do it or which examples I should look to. I'm interested in knowing whether it's possible to provide a list of target poses to cuRobo and have it generate plans to sequentially navigate through each one. Or is planning multiple poses a feature only available in MPC?

**Comment by balakumar-s** [comment, marked as answer]
You would have to write the outer loop to plan to reach a sequence of poses. Essentially a for-loop with each planning problem starting at the last time step of the previous trajectory.
